content/docs/studio/selfhosted/configuration/github.md

```diff
@@ -0,0 +1,75 @@
+# GitHub integration
+
```

Comment:
Should we say that this needs to be set up before installation? prep?

Reply:
https://github.com/iterative/dvc.org/pull/4344#discussion_r1124704404